duneanaobj/StandardRecord/SRTrack.h

```diff
@@ -22,6 +22,12 @@ namespace caf
 
       float Evis  = -999.;   ///< Visible energy in voxels corresponding to this track
 
+      // Track characteristics
+      float len_gcm3 = -999.; ///< Track length in g/cm3
+      float E = -999; ///< Track energy in MeV
```

Comment:
how does this differ from Evis above?

Reply:
It was meant to be the energy estimator using track length, whereas Evis I thought was the visible energy contained in the track?

Reply:
Ah, I see.  In the analysis I was doing with CAFs the analagous ND-LAr estimator is not baked into the files but rather computed from the track lengths stored in the CAFs.  I don't suppose it matters too much---we can leave this here---but for my edification is there any reason that couldn't be done in this case?